### PR TITLE
[feature]add workerId and starlet_port in showBackends

### PR DESCRIPTION
**Pull request by abc982627271**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/common/proc/BackendsProcDir.java [hunk 1]
```diff
@@ -52,11 +53,11 @@ public class BackendsProcDir implements ProcDirInterface {
 
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
             .add("BackendId").add("Cluster").add("IP").add("HostName").add("HeartbeatPort")
-            .add("BePort").add("HttpPort").add("BrpcPort").add("LastStartTime").add("LastHeartbeat").add("Alive")
-            .add("SystemDecommissioned").add("ClusterDecommissioned").add("TabletNum")
-            .add("DataUsedCapacity").add("AvailCapacity").add("TotalCapacity").add("UsedPct")
-            .add("MaxDiskUsedPct").add("ErrMsg").add("Version").add("Status").add("DataTotalCapacity")
-            .add("DataUsedPct").add("CpuCores").build();
+            .add("BePort").add("HttpPort").add("BrpcPort").add("StarletPort").add("WorkerId")
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/common/proc/BackendsProcDir.java:56]: The new ones should be added to last to ensure compatibility.

File: fe/fe-core/src/main/java/com/starrocks/catalog/StarOSAgent.java [hunk 2]
```diff
@@ -147,4 +147,14 @@ public void addWorker(long backendId, String workerIpPort) {
         workerToBackend.put(workerId, backendId);
         LOG.info("add worker {} success, backendId is {}", workerId, backendId);
     }
+
+    public long getWorkerIdfromBackendId(long backendId) {
+        long workerId = -1;
+        for (Map.Entry<Long, Long> entry : workerToBackend.entrySet()) {
+            if (entry.getValue() == backendId) {
+                workerId = entry.getKey();
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/catalog/StarOSAgent.java:155]: break

File: fe/fe-core/src/main/java/com/starrocks/common/proc/BackendsProcDir.java [hunk 3]
```diff
@@ -51,11 +52,11 @@ public class BackendsProcDir implements ProcDirInterface {
...
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
             .add("BackendId").add("Cluster").add("IP").add("HeartbeatPort")
-            .add("BePort").add("HttpPort").add("BrpcPort").add("LastStartTime").add("LastHeartbeat").add("Alive")
-            .add("SystemDecommissioned").add("ClusterDecommissioned").add("TabletNum")
+            .add("BePort").add("HttpPort").add("BrpcPort").add("LastStartTime").add("LastHeartbeat")
+            .add("Alive").add("SystemDecommissioned").add("ClusterDecommissioned").add("TabletNum")
             .add("DataUsedCapacity").add("AvailCapacity").add("TotalCapacity").add("UsedPct")
             .add("MaxDiskUsedPct").add("ErrMsg").add("Version").add("Status").add("DataTotalCapacity")
-            .add("DataUsedPct").add("CpuCores").build();
+            .add("DataUsedPct").add("CpuCores").add("StarletPort").add("WorkerId").build();
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/common/proc/BackendsProcDir.java:59]: Add the new names when integrate_starmgr is true.

File: fe/fe-core/src/test/java/com/starrocks/qe/ShowExecutorTest.java [hunk 4]
```diff
@@ -568,6 +573,47 @@ public void testShowColumnFromUnknownTable() throws AnalysisException, DdlExcept
...
+
+        Config.integrate_starmgr = true;
+        ShowBackendsStmt stmt = new ShowBackendsStmt();
+        ShowExecutor executor = new ShowExecutor(ctx, stmt);
+        ShowResultSet resultSet = executor.execute();
+
+        Assert.assertEquals(26, resultSet.getMetaData().getColumnCount());
+        Assert.assertEquals("BackendId", resultSet.getMetaData().getColumn(0).getName());
+        Assert.assertEquals("StarletPort", resultSet.getMetaData().getColumn(24).getName());
+        Assert.assertEquals("WorkerId", resultSet.getMetaData().getColumn(25).getName());
```
Comment (wyb) [inline fe/fe-core/src/test/java/com/starrocks/qe/ShowExecutorTest.java:614]: Set integrate_starmgr = false at last.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 22 / 28 (78.57%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/common/proc/BackendsProcDir.java|11|15|73.33%|
|:large_blue_circle:|com/starrocks/catalog/StarOSAgent.java|7|8|87.50%|
|:large_blue_circle:|com/starrocks/common/Config.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/system/Backend.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/system/HeartbeatMgr.java|1|1|100.00%|

**Review by starrocks-xupeng — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 26 / 28 (92.86%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|1|2|50.00%|
|:large_blue_circle:|com/starrocks/catalog/StarOSAgent.java|7|8|87.50%|
|:large_blue_circle:|com/starrocks/common/proc/BackendsProcDir.java|15|15|100.00%|
|:large_blue_circle:|com/starrocks/common/Config.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/system/Backend.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/system/HeartbeatMgr.java|1|1|100.00%|

**Review by wyb — APPROVED** [review]

**Review by sduzh — APPROVED** [review]

**Pull request merged**